dune/grid/common/ZoltanGraphFunctions.hpp

```diff
@@ -144,7 +145,11 @@ class CombinedGridWellGraph
     /// \brief Post process partitioning to ensure a well is completely on one process.
     /// \param[inout] parts The assigned partition numbers for each vertex.
     void postProcessPartitioningForWells(std::vector<int>& parts);
-    
+
+    double transmissibility(int face_index) const
+    {
+        return transmissibilities_ ? (1.0e18*transmissibilities_[face_index]) : 1;
```

Comment:
As I understand the current code, the weights are 1 for the faces, which will be replaced by this expression. The well weights are `std::numeric_limits<float>::max`, which makes me wonder: how is this dealt with in terms of scaling? Will it notice at all if one transmissibility is twice as large as another? Is the 1e18 factor something you have experimented with?